lib/widgets/message_list.dart

```diff
@@ -964,25 +963,20 @@ class MessageWithPossibleSender extends StatelessWidget {
           if (senderRow != null)
             Padding(padding: const EdgeInsets.fromLTRB(16, 2, 16, 0),
               child: senderRow),
-          Row(crossAxisAlignment: CrossAxisAlignment.start, children: [
-            const SizedBox(width: 16),
-            Expanded(
-              child: Column(
+          SwipableMessageRow(
+            message: message,
+            children: [
+              Expanded(child: Column(
                 crossAxisAlignment: CrossAxisAlignment.stretch,
                 children: [
                   MessageContent(message: message, content: item.content),
                   if ((message.reactions?.total ?? 0) > 0)
                     ReactionChipsList(messageId: message.id, reactions: message.reactions!)
                 ])),
-            SizedBox(width: 16,
-              child: message.flags.contains(MessageFlag.starred)
-                // TODO(#157): fix how star marker aligns with message content
-                // Design from Figma at:
-                //   https://www.figma.com/file/1JTNtYo9memgW7vV6d0ygq/Zulip-Mobile?node-id=813%3A28817&mode=dev .
-                ? Padding(padding: const EdgeInsets.only(top: 4),
-                    child: Icon(ZulipIcons.star_filled, size: 16, color: _starColor))
-                : null),
-          ]),
+              SizedBox(width: 16,
+                child: message.flags.contains(MessageFlag.starred)
+                  ? Icon(ZulipIcons.star_filled, size: 16, color: designVariables.star)
+                  : null)]),
```

Comment:
nit:
```suggestion
                  : null),
            ]),
```

(The pattern is that when there's a list, so the items are peers or siblings of each other, the last item gets a trailing comma just like the other items and the closing delimiter like `]` goes on the next line.

When there's one item that's the main payload and the other items are configuration around that — so in particular, for the `child` parameter vs. the other parameters of some widget constructor — then the main payload goes at the end and piles up the closing `)` delimiters without spilling onto more lines.)